Networking/Sources/Networking/Networking.swift

```diff
@@ -1,6 +1,33 @@
+import Alamofire
+import Foundation
+
 public struct Networking {
     public private(set) var text = "Hello, World!"
 
     public init() {
     }
+    
+    public func fetchAllProducts<T: Codable>(expectingReturnType: T.Type, completion: @escaping ((Result<T?,Error>) -> Void)){
+        guard let url = URL(string: Constans.baseUrl + DokanAPI.getAllProducts.path) else { return }
```

Comment:
Return without calling the completion will keep the closure wait without response.. You may not be able to know that this method is failing as well.

Reply:
Please raise this point so we can discuss it in the session.